prow/cmd/hook/main.go

```diff
@@ -110,34 +108,30 @@ func main() {
 		logrus.WithError(err).Fatal("Error starting config agent.")
 	}
 
+	var tokens []string
+	var err error
+
+	// Append the path of hmac and github secrets.
+	tokens = append(tokens, o.githubTokenFile)
+	tokens = append(tokens, o.webhookSecretFile)
+
+	// This is necessary because there is no default value for slack token file.
```

Comment:
technically necessary since the slack client is optional (which is why there is no default, unlike for the other tokens, which are mandatory)